src/util-dpdk-bonding.c

```diff
@@ -52,7 +52,10 @@ uint16_t BondingMemberDevicesGet(
 {
 #ifdef HAVE_DPDK_BOND
 #if RTE_VERSION >= RTE_VERSION_NUM(23, 11, 0, 0)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Why do we need this ? Why cannot we have a non deprecated declaration ?

Reply:
Non deprecated declaration is not possible as DPDK in 23.11 marked the old API (slave) as deprecated and the new API (member) as experimental. On both versions of this API therefore a warning is thrown. I circumvented this by allowing experimental API in the Github Actions.